examples/cactus-example-supply-chain-business-logic-plugin/src/main/typescript/business-logic-plugin/web-services/insert-shipment-endpoint-constants.ts

```diff
@@ -0,0 +1,8 @@
+export class InsertShipmentEndpoint {
```

Comment:
@jordigiam These constants classes will disappear completely eventually in favor of pulling the values directly from the openapi.json files. (just FYI, I'm okay to accept this as is for now)

Reply:
Done